test/clj/rems/api/test_catalogue_items.clj

```diff
@@ -90,6 +97,13 @@
                              handler
                              read-ok-body)]
             (is (:success response) (pr-str response))
+            (testing "application is updated when catalogue item is edited"
+              (let [app-id (test-data/create-application! {:catalogue-item-ids [id]
+                                                           :actor "alice"})
+                    get-app #(applications/get-unrestricted-application app-id)]
+                (is (= {:fi "http://info.fi"}
```

Comment:
For some reason, this test doesn't fail even when the cache is not cleared in rems.api.services.catalogue, not sure why.

Reply:
Fixed, I had a bug in the test (it re-created the application after editing catalogue item).